Explorer/Assets/DCL/SDKComponents/MediaStream/Wrapper/MediaPlayerPluginWrapper.cs

```diff
@@ -75,6 +75,8 @@ public void InjectToWorld(ref ArchSystemsWorldBuilder<World> builder, ISceneData
             CleanUpMediaPlayerSystem.InjectToWorld(ref builder, mediaPlayerPool, videoTexturePool);
 
             VideoEventsSystem.InjectToWorld(ref builder, ecsToCrdtWriter, sceneStateProvider, componentPoolsRegistry.GetReferenceTypePool<PBVideoEvent>(), frameTimeBudget);
+
+            finalizeWorldSystems.RegisterReleasePoolableComponentSystem<MediaPlayer, MediaPlayerComponent>(ref builder, componentPoolsRegistry);
```

Comment:
I think it's not enough as Video textures can leak in the similar fashion, I propose you to make `CleanUpMediaPlayerSystem` implement `IFinalizeWorldSystem` and handle everything there in one place as there is already the code for both MediaPlayer and VideoTextures